docs/kerala/getting-started.md

```diff
@@ -38,35 +38,16 @@ $ git clone git@github.com:simpledotorg/simple-server.git
 
 ### 1. Set up your AWS account
 
-- Create an IAM user group in the new AWS account called `Provisioners` with the following policies (`My Security Credentials` > `Groups` > `Create new group`)
```

Comment:
We should use our own IAM keys for using terraform (we do this already). Having a shared user means the same key needs to be shared which can be a pain to revoke later.